src/ga.jl

```diff
@@ -122,8 +122,8 @@ function ga(objfun::Function, N::Int;
 
         # Terminate:
         #  if fitness tolerance is met for specified number of steps
-        if fittol < tol
-            if fittolitr > tolIter
+        if fittol <= tol
+            if (tolIter != -1) && (fittolitr > tolIter)
```

Comment:
If you set  `tolIter` to -1, algorithm would run for number of integrations specified in `iterations`. What is the point in it?
You can just increase tolerance. I would rather  add assertion that `tolIter` must be positive.

Reply:
There needs to be a value for tolIter which represents turning it off, or some other mechanism for this.

Reply:
The other mechanism would be increase tolerance. This parameter specifies an additional number of iteration when provided tolerance is achieved. Usually, the algorithm can quit at this point, but it will run some more iterations. So, this parameter should be always positive otherwise it does not make sense to use it.

Reply:
I guess the user could set tolIter = iterations easily enough to turn off early stopping. I'll remove the -1 option.